spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -147,6 +148,11 @@ public static void configureServerWithArmeriaSettings(ServerBuilder server, Arme
                     DropwizardSupport.addExposition(settings, server, meterRegistry);
                 }
             }
+
+            server.decorator(MetricCollectingService.newDecorator(
+                    MeterIdPrefixFunction.ofDefault("armeria.server")
+                                         .andThen((registry, log, meterIdPrefix) -> meterIdPrefix.withTags(
+                                                 "service", log.serviceName()))));
```

Comment:
You don't need to set `service` tag. :-)
https://github.com/line/armeria/blob/7f3295f32ac263ef83d96e6c763c1395431681b9/core/src/main/java/com/linecorp/armeria/common/metric/MeterIdPrefixFunction.java#L108-L111

Reply:
I think we should remove the following method to avoid duplicate collecting.
https://github.com/line/armeria/blob/8c9c81f30683fd44cec2f629fd8d543b8ad43d21/spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java#L339-L349

Reply:
Oops, I missed this comment. 😄